README.md

```diff
@@ -239,7 +243,7 @@ The following environment variables can be passed to the container:
 
 | Name                               | Description                                                  | Default value     | Required |
 | ---------------------------------- | ------------------------------------------------------------ | ----------------- | -------- |
-| PATHFINDER_ETHEREUM_API_URL        | Ethereum (archive) node JSON-RPC endpoint URL                |                   | yes      |
+| PATHFINDER_ETHEREUM_API_URL        | Ethereum full node JSON-RPC endpoint URL                |                   | yes      |
```

Comment:
Is it just the GitHub diff, or are we missing some extra whitespace in the new line?

Reply:
Ah yeah you're right. Fixed.